cmake/modules/StandaloneOverlay.cmake

```diff
@@ -9,6 +9,12 @@ if(DEFINED SWIFT_MASTER_LOADED
 endif()
 
 
+list(APPEND CMAKE_MODULE_PATH
+  "${SWIFT_SOURCE_ROOT}/llvm/cmake/modules"
```

Comment:
There is no guarantee that `SWIFT_SOURCE_ROOT` is defined.  I think it might be better to rely on the standard behaviour of CMake:

```
find_package(LLVM CONFIG REQUIRED)
```

That will error out if the user does not specify `LLVM_DIR` or does not have LLVM installed and available on the system.

Reply:
In all cases where this module is currently used it should be safe to assume that `SWIFT_SOURCE_ROOT` is defined. I hope to remove the need for this extra system in the near future (should it be possible) then we won't have to worry about this odd inconsistency.